### ISSUE TITLE
Display current gcode line somewhere.

### ISSUE THREAD

**Issue by darenschwenke** [qtquickvcp/Machineface#9]
When used for 3D printing, knowing the gcode line currently executing can be useful.
If a 3D print fails, and you happen to be around to catch it, you stop the print.
Then if you knew the line it failed on or close to it, the print could be resumed by creating a new gcode file and deleting all lines before that.

Perhaps clicking on (for tablet) or click/hover (for mouse) of the percent complete bar could display the current line.

**Comment by machinekoder** [comment]
Good idea. The current line should be straight-forward to access in the progress bar.
